floorplan-cli/build.gradle

```diff
@@ -17,25 +17,26 @@ application {
     getMainClass().set("com.zynger.floorplan.FloorPlanCliKt")
 }
 
-tasks.register('embarkVersionFile') {
+def embarkVersionFile = tasks.register('embarkVersionFile') {
```

Comment:
just a nit: the `embarkVersionFile` variable is actually a task provider not a file. maybe naming it `embarkVersionTask` or `embarkVersionTaskProvider` may clarify.